tests/test_base_client.py

```diff
@@ -239,3 +239,15 @@ def test_query_param_encoding(self):
 
                 request.assert_called_with("GET", url, None, headers=expected_headers)
                 self.assertEqual(type(resource), MyResource)
+
+    def test_client_can_set_timeout(self):
+        timeout = 3
+        client = MockClient("apikey", timeout=timeout)
+        self.assertEqual(client.timeout, timeout)
+
+    def test_client_timeout(self):
+        client = MockClient("apikey", timeout=0)
+        with self.assertRaises(recurly.NetworkError) as e:
+            client._make_request("GET", "wef", body=None, params=None)
```

Comment:
It's kind of dangerous to make a real call here, but also the most realistic to make sure timeout is working. I'm definitely open to ways to make this better or fit with the testing paradigm for this project.

Reply:
I've removed this test in order to appease the `guardrails` bot

Reply:
Hmmm, looks like `guardrails` is still grumpy about the code, so maybe this wasn't the culprit. I'll wait to make any further changes until I get more guidance on what actually needs to be fixed.